multiregion/README.md

```diff
@@ -124,11 +125,12 @@ The script creates each topic with a different replica placement policy that spe
 The replica placement policy file is defined with the argument `--replica-placement <path-to-replica-placement-policy-json>` mentioned earlier (these files are in the [config](config/) directory).
 Each placement also has a minimum `count` associated with it that allows users to guarantee a certain spread of replicas throughout the cluster.
 
-| Topic name         | Leader  | Followers (sync replicas) | Observers (async replicas) | ISR list  |
-|--------------------|---------|---------------------------|----------------------------|-----------|
-| single-region      | 1x west | 1x west                   | n/a                        | {1,2}     |
-| multi-region-sync  | 1x west | 1x west, 2x east          | n/a                        | {1,2,3,4} |
-| multi-region-async | 1x west | 1x west                   | 2x east                    | {1,2}     |
+| Topic name           | Leader  | Followers (sync replicas) | Observers (async replicas) | ISR list  |
+|----------------------|---------|---------------------------|----------------------------|-----------|
+| single-region        | 1x west | 1x west                   | n/a                        | {1,2}     |
+| multi-region-sync    | 1x west | 1x west, 2x east          | n/a                        | {1,2,3,4} |
+| multi-region-async   | 1x west | 1x west                   | 2x east                    | {1,2}     |
+| multi-region-default | 1x west | 1x west                   | 2x east                    | {1,2}     |
```

Comment:
From this table, `multi-region-async` looks identical to `multi-region-default`.  Users will be confused as to what the difference is.  Can you please add a column that represents the difference?